SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,48 @@
+	described by materialInputs ("Material inputs for this process step") with values of type MaterialInputs,
+	described by processSimulation (note "Reference to Simulation records associated with this processStep.") with values of type ProcessSimulation.
+
+ManufacturingSpecification (note "A specification for a manufacturing process") is a type of ProcessStepSpecification,
+	described by globalProcessParameters (note "Nominal system setting which apply to all material stock involved in the Manufacturing Process Step." ) with value of type GlobalProcessParameters,
+	described by partSpecificProcessParameters (note "Nominal system setting which uniquely apply to an individual part to be processed." ) with values of type PartSpecificProcessParameters,
+ 	described by specifiedEnvironment with a single value of type Environment.
+
+// Suggested 'SpecifiedQuality' with upper and lower bounds
+Environment is a class,
+	described by ambientTemperature with a single value of type SpecifiedQuantity,
+	described by ambientPressure with a single value of type SpecifiedQuantity.
+
+GlobalProcessParameters is a class.
+
+PartSpecificProcessParameters is a class,
+	described by orientation (note "Specified orientation for the specific part." ) with a single value of type Orientation,
+	described by design (note "design file which prescribes the changes to the part which are specific to this process." ) with a single value of type Design.
+
+Consumables (note "Materials which are used during the execution of a process, but do not, in a substantial way, become part of material output. Example: Process Gas") is a class,
+
+MaterialInputs (note "Materials that are an input to the process and becomes a part of the material output in a substantial way. Examples: Powder Feedstock, billet, sub-components to be assembled.") is a class.
+
+ProcessSimulation (note "Data used and generated from computer programs using physics based models: including the models and the simulation outputs.") is a class,
+	described by simulationID (note "UID for specific simulation.") with a single value of type string,
+	described by simulationModel (note "Physical phenomenon covered, mathematical equations.") with a single value of type Document,
+	described by simulationSoftware (note "Software information.") with values of type Software,
+	described by simulationInputs (note "Parameters used for simulations.") with a single value of type Document,
+	described by simulationOutputs (note "Simulation results.") with a single value of type Document,
+	described by modelDomain (note "Focal area of model.") with values of type string,
+	described by modelDefinition (note "Documentation of how model is constructed (equations, I/O).") with a single value of type Document.
+
+ManufacturingCreationProcessStep (note "Manufacturing process in which a new material stock or part is created" ) is a type of ManufacturingProcessStep,
```

Comment:
add something to the note saying that a material has been transformed?

Reply:
Assigned to @alkitt